CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## Release 1.3.0-SNAPSHOT (2021-09-14)
+
```

Comment:
Since this PR fixes the nexus upload and does not introduce a new feature i'd argue that it's release version is 1.2.1. (This should also be adapted in the branch and PR name itself)
```suggestion
## Release 1.2.1 (2021-10-14)
- Fix automatical deployment to nexus repository upon release for development and master branch (PR #48)
```